iroh-bytes/src/protocol.rs

```diff
@@ -284,3 +298,17 @@ impl TryFrom<VarInt> for Closed {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn test_handshake_ser() {
+        let handshake = Handshake::new();
+        let ser = postcard::to_stdvec(&handshake).unwrap();
+        assert_eq!(ser.len(), HANDSHAKE_SIZE);
```

Comment:
The postcard format is stable, so we could actually assert the byte(s) here as well, right?